### PR TITLE
add missing value function, tests, dependencies

### PR DESCRIPTION
**Pull request by andrealee011**

### CODE REVIEW

**Comment by andrealee011** [comment]
- deleted `pyb4model.egg-info` folder that should not be commited to github
- add `test_fit_and_report` function to main test file `test_pyb4model.py` and delete `test_fit_and_report.py` file
- add missing value function, missing value test function, and its dependencies

File: pyb4model/pyb4model.py [hunk 1]
```diff
@@ -24,10 +30,37 @@ def missing_val(df, method):
...
+    
+    # function
+    
+    if method=='delete':
+        df = df.dropna()
+    
+    if method=='mean':
+        df = df.fillna(df.mean())
+        
+    if method=='knn':
```
Comment (xiw315) [inline pyb4model/pyb4model.py:57]: This is not an error, just an suggestion for something we can implement in the future if we have time: we could accept argument that is case insensitive. 
```suggestion
    if method.lower()=='delete':
        df = df.dropna()
    
    if method.lower()=='mean':
        df = df.fillna(df.mean())
        
    if method.lower()=='knn':
```

**Review by xiw315 — APPROVED** [review]
Thanks for deleting the unnecessary files! Didn't spot any bug or error. Just add one suggestion on things that we could implement in the future

File: pyb4model/pyb4model.py (hunk 1 above)
Comment (andrealee011) [reply pyb4model/pyb4model.py:57]: great idea! i will definitely implement this !

**Pull request merged**